fanficfare/adapters/adapter_spiritfanfictioncom.py

```diff
@@ -0,0 +1,350 @@
+        summary = div_element.find('div', class_='texto')
+
+        text = summary.find('strong').next_sibling
+        full_text = ''
+        while text is not None:
+            if text.name == 'br':
+                if full_text != '':
+                    full_text += '<br/>'
+                else:
+                    full_text += '<p>'
+            elif text.name == 'a' and text.has_attr('data-cfemail'):
+                # Ignore obfuscated email link and its associated text
+                pass
+            else:
+                full_text += text.strip()
+            text = text.next_sibling
+        full_text += '</p>'
+        self.story.setMetadata('description', full_text)
+        
+        
+        def parse_until_br(attribute, start_index, element_list):
+            # Initialize counter
+            next_index = start_index
+            for element in element_list[start_index:]:
+                next_index += 1
+                if element.name == 'br':
+                    break
+                elif element.name == 'strong':
+                    if attribute == 'status':
+                        if element.contents[0].text == 'Sim':
+                            self.story.setMetadata(attribute, 'Completed')
+                        elif element.contents[0].text == 'Não':
+                            self.story.setMetadata(attribute, 'In-Progress')
+                    elif attribute == 'characters':
+                        terms = re.findall(r"[^&]+", element.text.strip())
+                        for term in terms:
+                            self.story.addToList(attribute, term.strip())
+                    elif attribute == 'warnings':
+                        self.story.addToList(attribute, element.text.strip())
+                    elif attribute == 'numWords':
+                        self.story.setMetadata(attribute, element.text.replace('.','').strip())
+                    else:
+                        self.story.setMetadata(attribute, element.text.strip())
+                elif element.name == 'a':
+                    if element.contents[0].name == 'strong':
+                        self.story.addToList(attribute, element.contents[0].text.strip())
+                elif element.name == 'time':
+                    self.story.setMetadata(attribute, makeDate(element['datetime'], self.dateformat))
+            return next_index
+
+        # Informações Gerais
+        information = div_element.find(lambda tag: tag.name == 'div' and
+                                       tag.get('class') == ['texto', 'espacamentoTop'] and
+                                       tag.get('id') != 'cphConteudo_cphConteudo_divDestaque')
+        logger.debug('information: (%s)', information)
+        info_contents = information.contents
+        #for i, content in enumerate(info_contents):
+        i = 0
+        while i < len(info_contents):
+            content = info_contents[i]
+            if content.text.strip() == 'Iniciado':
```

Comment:
I'd use a `dist()` linking the content title to the metadata entry instead of a long if/elif, but that's more personal choice.